boards/arm/gd32f403z_eval/gd32f403z_eval.dts

```diff
@@ -26,3 +39,13 @@
 	pinctrl-0 = <&usart0_default>;
 	pinctrl-names = "default";
 };
+
+&timer0 {
+	status = "okay";
+
+	pwm0: pwm {
```

Comment:
Suggestion:

```diff
&timer0 {
	status = "okay";
+	prescaler = <256>;

	pwm0: pwm {
```

Reply:
thanks for the feedback. I'll try on a timer0/7 (advanced) on my board to see if it works.